addons/http_routing/models/ir_http.py

```diff
@@ -223,8 +238,9 @@ def _get_converters(cls):
         )
 
     @classmethod
-    def _get_language_codes(cls):
-        return request.env['res.lang'].get_installed()
+    @ormcache()
```

Comment:
You should [not return a recordset](https://github.com/odoo/odoo/blob/8e0dd5f52fb876ce8133007d97b464c49188cca6/odoo/tools/cache.py#L48) in an ormcache. We used `ormcache` on `_lang_get_id` and makes a browse in `_lang_get` for that reason (otherwise we get errors like reusing a closed cursor).

Reply:
Thank you, I was not fully aware of that, I ensured it was working since it felt weird but despite working, I did not know about the closed cursor thingy.

I removed that, since it will now be in base I don't need that anymore anyway.